src/components/swap/SwapPlatformSelector.tsx

```diff
@@ -38,6 +45,24 @@ interface GasFeeProps {
   gasFeeUSD: CurrencyAmount | null
 }
 
+const StyledFlex = styled(Flex)`
+  gap: 4px;
+`
```

Comment:
Shouldn't `Flex` support gap prop? [`gap` is in the specs](https://css-tricks.com/almanac/properties/g/gap/)

```suggestion
const StyledFlexWithGap = styled(Flex)`
  gap: 4px;
`
```

Reply:
Rebass Flex doesn't have gap prop on its types...